<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVPropBuildingZoneProperty</class>
 <widget class="QWidget" name="SVPropBuildingZoneProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Building:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxBuildingFilter"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxBuildingLevelFilter"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Building level:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Room Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNameFilter"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Rooms/Zones:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableViewZones">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxSelectedComponent2">
       <property name="title">
        <string>Update Floor Areas / Volumes</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Only selected rooms:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonFloorAreaSelectedRooms">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Floor area</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonVolumeSelectedRooms">
            <property name="text">
             <string>Volume</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>All rooms:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonFloorAreaAllRooms">
            <property name="text">
             <string>Floor area</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonVolumeAllRooms">
            <property name="text">
             <string>Volume</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Other</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButtonAssignSurface">
          <property name="text">
           <string>Assign selected surfaces to room</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
